react/styles.css

```diff
@@ -1,11 +1,19 @@
 .backdropContainer {
+  top: 0;
+  left: 0;
```

Comment:
why not use tachyons instead?

Reply:
These styles are necessary to the component to work and it should not be changed in any circumstances. Another point is that not all styles that I'm applying are available in tachyons, so I would have all these styles that are meant to work together in different files.